odoo/addons/test_pylint/tests/test_pylint.py

```diff
@@ -115,11 +138,11 @@ def test_pylint(self):
             '--deprecated-modules=%s' % ','.join(self.BAD_MODULES)
         ]
 
-        try:
-            process = subprocess.Popen(['pylint'] + options + paths, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        except (OSError, IOError):
-            self._skip_test('pylint executable not found in the path')
-        else:
-            out, err = process.communicate()
-            if process.returncode:
-                self.fail("\n" + out + "\n" + err)
+        with _patch_streams(StringIO()):  # Avoid show 'no config file msg'
```

Comment:
This should not be necessary [if you use the `pylint.lint.PyLinter` class directly](https://github.com/odoo/odoo-extra/pull/94/files#diff-2a21db3452219209e5224d39432c7c1dR34), though you will have to perform the actions of the various options.